### DISCUSSION TITLE
QNAP 装了 Alist Docker, 但是 Log 里面没有用户名和密码

### DISCUSSION THREAD

**Discussion by agentictaskx** [AlistGo/alist#5240] [Q&A]
尝试了 admin/admin, admin/123456, admin/;    都不能登录，也没有 Terminal 去看密码，安装的时候不能用命令行强行设置一个密码吗？

**Comment by anwen-anyi** [comment]
Docker安装的? 自己去看文档有告诉你怎么获取密码

**Comment by richardshi10** [comment]
解决了没有 一样的问题

**Comment by kasimwu** [comment]
我解决了，我猜没密码的人都安装了较高的版本，因为3.25版本后密码是hash随机值，只能从后台获取，所以原本的密码失效了。我注意到qnap帖子上写的是3.18版本，然后我卸载了安装的新版本，装上3.18后原本的密码起作用了。希望能帮到你
